staramr/blast/plasmidfinder/PlasmidfinderBlastDatabase.py

```diff
@@ -12,12 +14,52 @@
 
 class PlasmidfinderBlastDatabase(AbstractBlastDatabase):
 
-    def __init__(self, database_dir: str) -> None:
+    def __init__(self, database_dir: str, database_type: str = None) -> None:
         """
         Creates a new PlasmidfinderBlastDatabase.
-        :param database_dir: The specific ResFinder database (drug class) directory.
+        :param database_dir: The specific PlasmidFinder database directory.
         """
+
         super().__init__(database_dir)
+        self.plasmidfinder_database_dir = path.join(self.database_dir)
```

Comment:
I don't think this variable is used anywhere is it? If not, could it be removed?

Reply:
It used to be but now it isn't, I removed it completely in commit 48f2a20